### PR TITLE
Add AKShare datasource for PyBroker

### PR DESCRIPTION
**Pull request by jindaxiang**
Hi Ed,
I have added AKShare datasource for PyBroker, please check it in PyBroker dev branch!
I have tested in my computer, and it works fine just like yfinance, but now we can just fetch chinese stock data, I will put more features to  AKShare datasource in the future.
1. add AKShare info to README.md
2. add AKShare to data.py
3. add AKShare to requirements.txt
4. add AKShare to docs/Getting Started with Data Sources.ipynb
5. add AKShare to setup.cfg
Can you add some test to tests/test_data.py?

### CODE REVIEW

File: src/pybroker/data.py [hunk 1]
```diff
@@ -532,3 +533,98 @@ def _fetch_data(
...
+        self._scope.register_custom_cols(self.ADJ_CLOSE)
+
+    def query(
+        self,
+        symbols: Union[str, Iterable[str]],
+        start_date: Union[str, datetime],
+        end_date: Union[str, datetime],
+        _: Optional[str] = "",
+    ) -> pd.DataFrame:
+        r"""Queries data from `Yahoo Finance <https://finance.yahoo.com/>`_\ .
```
Comment (edtechre) [inline src/pybroker/data.py:559]: Can you update the docstring to AKShare?

File: src/pybroker/data.py [hunk 2]
```diff
@@ -532,3 +533,98 @@ def _fetch_data(
...
+            '日期': DataCol.DATE.value,
+            '开盘': DataCol.OPEN.value,
+            '收盘': DataCol.CLOSE.value,
+            '最高': DataCol.HIGH.value,
+            '最低': DataCol.LOW.value,
+            '成交量': DataCol.VOLUME.value,
+            "symbol": DataCol.SYMBOL.value,
+
+        }, inplace=True)
+        result['date'] = pd.to_datetime(result['date'])
```
Comment (edtechre) [inline src/pybroker/data.py:619]: The formatting will fail the tests that run post-commit. Can you run these commands?

1. black (auto format)
2. flake8 (lint)
3. mypy --show-error-codes --ignore-missing-imports (typecheck)

You can install these with:
pip install black flake8 flake8-bugbear mypy

**Review by edtechre — CHANGES_REQUESTED** [review]
Thank you for this, Albert!

File: src/pybroker/data.py (hunk 2 above)
Comment (jindaxiang) [reply src/pybroker/data.py]: ok

**Review by jindaxiang — COMMENTED** [review]
fix it

**Review by edtechre — APPROVED** [review]

**Pull request merged**